config/application.rb

```diff
@@ -22,5 +22,7 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    config.action_view.field_error_proc = Proc.new { |html_tag, instance| html_tag }
```

Comment:
In questions _form partial we have radio-buttons hided into labels with 'btn' class. When some field in form was not filled bootstrap cover labels into div with some error_class and radio-buttons became ugly. This config disable this feature.